shared/gcp_jupyterlab_shared/version.py

```diff
@@ -12,4 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-VERSION = '1.0.11'
+VERSION = '1.0.13'
```

Comment:
Any reason we are bumping it by 2 minor versions?

Reply:
I think I forgot to at some point change this file, package.json was at 1.0.13

Reply:
Ok. I was just wondering what happened to 1.0.12 but then I realized that it was a previous change in this branch that is not yet on master.